modules/apps/accessibility/accessibility-test/src/testFunctional/tests/AccessibilityMenu.testcase

```diff
@@ -36,90 +36,68 @@ definition {
 		}
 	}
 
-	@description = "LPS-178192. Verifies only if the setting has never been changed by the user, either when signed in or not signed in, will the default value be displayed.."
+	@description = "LPS-178192."
 	@priority = 5
-	test BrowserLocalStorageSmoke {
+	test CanBeDisabledByInstanceSettings {
 		property portal.acceptance = "true";
 
-		task ("Given created new user A") {
-			JSONUser.addUser(
-				userEmailAddress = "userea@liferay.com",
-				userFirstName = "userfnA",
-				userLastName = "userlnA",
-				userScreenName = "usersnA");
-		}
-
-		task ("And signed out as guest user") {
-			SignOut.signOut();
+		task ("Given user with admin privileges") {
+			Navigator.openURL();
 		}
 
-		task ("When the toggle shows underline to ON state") {
-			AccessibilityMenu.enableSpecificOption(key_toggleSwitchLabel = "Underlined Links");
-
-			AssertChecked.assertCheckedNotVisible(
-				key_toggleSwitchLabel = "Underlined Links",
-				locator1 = "ToggleSwitch#ANY_TOGGLE_SWITCH");
+		task ("When turn off Enable Accessibility Menu") {
+			AccessibilityMenu.deactivateAccessibilityMenu(portlet = "Instance Settings");
```

Comment:
This should be deactivateAccessibilityMenu, meaning it unchecks the config. We are not evaluating Reset Default Values here...